manifests/g/GanttProject/GanttProject/3.2/GanttProject.GanttProject.locale.en-US.yaml

```
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=QUSU.7-2-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: GanttProject.GanttProject
+PackageVersion: "3.2"
+PackageLocale: en-US
+Publisher: GanttProject
```

```suggestion
Publisher: BarD Software s.r.o.
```
This must match Control Panel.